tasks/connectors/snyk_v2/snyk_v2_task.rb

```diff
@@ -23,12 +23,12 @@ def self.metadata
               type: "date",
               required: false,
               default: 30,
-              description: "default will be 30 days before today format: YYYY-MM-DD" },
+              description: "Default will be 30 days before today format: YYYY-MM-DD" },
```

Comment:
This is not used as a date in the code, it should be defined as integer and the description should make it clear that it is meant to be a number of days.
https://github.com/KennaSecurity/toolkit/blob/980102c61ecd1bfb4706df64ef499b252ea63b51/tasks/connectors/snyk_v2/snyk_v2_task.rb#L197